layouts/joomla/form/field/textarea.php

```diff
@@ -42,16 +45,29 @@
  * @var   array    $options         Options available for this field.
  * @var   array    $inputType       Options available for this field.
  * @var   string   $accept          File types that are accepted.
+ * @var   boolean  $charcounter     Does this field support a character counter?
  */
 
 // Initialize some field attributes.
 $autocomplete = !$autocomplete ? 'autocomplete="off"' : 'autocomplete="' . $autocomplete . '"';
 $autocomplete = $autocomplete === 'autocomplete="on"' ? '' : $autocomplete;
 
+if ($charcounter)
+{
+	// Load the js file
+	Factory::getDocument()->getWebAssetManager()->enableAsset('short-and-sweet');
+
+	// Set the css class to be used as the trigger
+	$charcounter = 'charcount';
+
+	// Set the text
+	$counterlabel = 'data-counter-label="' . Text::_('COM_CONFIG_METADESC_COUNTER') . '"';
```

Comment:
Needs a leading space or it'll run into the attribute before it.
```suggestion
	$counterlabel = ' data-counter-label="' . Text::_('COM_CONFIG_METADESC_COUNTER') . '"';
```

Reply:
Err, nevermind, just re-read the smart code at the end.